nuxt.config.js

```diff
@@ -84,7 +84,7 @@ export default {
   css: [],
```

Comment:
I don't think the i18n files are supposed to be included in the MR. 

@ann-kilzer or @theyokohamalife can you confirm?

Reply:
It's okay to bump them in the PR only if there were corresponding changes, but we should update them to the latest before merging.